src/Conversions_TEST.cc

```diff
@@ -237,6 +238,7 @@ TEST(Conversions, Material)
   EXPECT_EQ(math::Color(1.3f, 1.4f, 1.5f, 1.6f),
       msgs::Convert(materialMsg.emissive()));
   EXPECT_TRUE(materialMsg.lighting());
+  EXPECT_DOUBLE_EQ(2.5, pbrMsg.render_order());
```

Comment:
This is failing to compile

```suggestion
  EXPECT_DOUBLE_EQ(2.5, materialMsg.render_order());
```